.github/workflows/enhanced-testing.yml

```
@@ -0,0 +1,529 @@
+            "include": [
+              {
+                "name": "Unit Tests - Express API",
+                "package": "express-api",
+                "test-pattern": "src/**/*.test.ts",
+                "exclude-pattern": "**/*.integration.test.* **/*.e2e.test.*",
+                "timeout": 120000,
+                "parallel": true
+              },
+              {
+                "name": "Unit Tests - Client UI",
+                "package": "client-ui", 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)